corrector.rb

```diff
@@ -0,0 +1,9 @@
+class Corrector
+  def correct_name(name)
+    name = name.capitalize
+    return name.slice(0...10) unless name.length < 10
```

Comment:
- [ ] Your method returns `nil` for names less than 10 in character length, whereas it should return a capitalized name anyways, despite being less than 10 in length. Kindly fix that.

  ```
  name_one.correct_name('ddddd')
  #=> Ddddd
  ```




  ![meri-corr](https://user-images.githubusercontent.com/36542986/156351808-e27325a6-1264-4376-9c83-bac743848cbd.jpg)